### DISCUSSION TITLE
Dashboard Design Outdated

### DISCUSSION THREAD

**Discussion by Muhammad-Arsal** [nasirkhan/laravel-starter#459] [Ideas]
Well you have a great starter kit. But what seems to be the problem is that the design of the admin panel is a bit outdated type. Don't you think it should be replaced with the modern design

**Comment by nasirkhan** [comment]
Thank you for using Laravel Starter. 

Yes, backend theme is a little outdated and I really want to use a better theme. But I could not manage enough time to look into new themes for Laravel Starter.

You may suggest new theme options which are free and have good documentation.

**Comment by Muhammad-Arsal** [comment]
Well there are plenty of themes out there based on bootstrap since you are using it. Theme Wagon got a plenty of themes that are free and also they are of latest design that will appeal


Here are some of the suggestions:

1.https://themewagon.com/themes/star-admin-2-free-bootstrap-4-html5-admin-dashboard-template/

2.https://github.com/puikinsh/Adminator-admin-dashboard

3.https://github.com/codescandy/Dash-UI

**Comment by cgdmohamed** [comment]
Hello there, i know it might an old topic, but can you ref the current admin panel ?

**Reply by nasirkhan** [reply]
CoreUI theme has been used in the Admin Backend: https://coreui.io/product/free-bootstrap-admin-template/
